src/mongo_logger.ts

```diff
@@ -285,6 +281,40 @@ function compareSeverity(s0: SeverityLevel, s1: SeverityLevel): 1 | 0 | -1 {
   return s0Num < s1Num ? -1 : s0Num > s1Num ? 1 : 0;
 }
 
```

Comment:
Are we missing a `LoggableCommandStartedEvent`?

Reply:
No, the Events and Logging Spec do not differ for `CommandStartedEvent`, so they both use one class defined in [src/cmap/command_monitoring_events.ts](https://github.com/mongodb/node-mongodb-native/blob/b93d405275c3a8ce6b1eba0af335ffb2a309e34e/src/cmap/command_monitoring_events.ts#L18)